### ISSUE TITLE
Kanister does not find Secret without "kind" in reference

### ISSUE THREAD

**Issue by gouglhupf** [kanisterio/kanister#219]
I was trying out Kanister on my local machine with Docker Desktop for Windows I stumbled over the following issue:
When omitting `kind: Secret` in the object reference of the postgres secret I get the following error (with "kind" key and value it works totally fine):
```
Failed to execute phase: v1alpha1.Phase{Name:"takeDatabaseBackup", State:"pending", Output:map[string]interface {}(nil)}: template: config:16:29: executing "config" at <.Secrets.postgres.Da...>: map has no entry for key "postgres"
github.com/kanisterio/kanister/pkg/param.renderStringArg
	/go/src/github.com/kanisterio/kanister/pkg/param/render.go:101
github.com/kanisterio/kanister/pkg/param.render
	/go/src/github.com/kanisterio/kanister/pkg/param/render.go:32
github.com/kanisterio/kanister/pkg/param.render
	/go/src/github.com/kanisterio/kanister/pkg/param/render.go:36
github.com/kanisterio/kanister/pkg/param.RenderArgs
	/go/src/github.com/kanisterio/kanister/pkg/param/render.go:18
github.com/kanisterio/kanister/pkg.(*Phase).Exec
	/go/src/github.com/kanisterio/kanister/pkg/phase.go:44
github.com/kanisterio/kanister/pkg/controller.(*Controller).runAction.func1
	/go/src/github.com/kanisterio/kanister/pkg/controller/controller.go:382
github.com/kanisterio/kanister/vendor/gopkg.in/tomb%2ev2.(*Tomb).run
	/go/src/github.com/kanisterio/kanister/vendor/gopkg.in/tomb.v2/tomb.go:163
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1333
```
### Cutout of the objects I deployed:
```yaml
apiVersion: cr.kanister.io/v1alpha1
kind: Blueprint
        ...
        command:
          - bash
          - -o
          - errexit
          - -c
          - |
...
            PGPASSWORD="{{ index .Secrets.postgres.Data "postgresql-password" | toString }}"
...
```
```yaml
apiVersion: cr.kanister.io/v1alpha1
kind: ActionSet
...
    secrets:
      postgres:
        # kind: Secret - works with this line
        name: postgres-secret
        namespace: default
```
### Software Versions
Kanister 0.20.0
Docker Desktop 2.1.0.1
Kubernetes 1.14.3

**Comment by gouglhupf** [comment]
The main problem / side effect is that I can't use `kanctl`. As far as I understand it is not possible to specify the object "kind" with `kanctl`

**Comment by vkamra** [comment]
@gouglhupf - thanks for reporting this and the `Blueprint`/`ActionSet` snippets.

`kind: Secret` is redundant information so we should be able to make it available in TemplateArgs even if it wasn't specified. I'll take a look at the logic there and follow up.

**Comment by vkamra** [comment]
@gouglhupf - Fixed with #222 . 

Validated using the following:
```
apiVersion: cr.kanister.io/v1alpha1
kind: Blueprint
metadata:
  name: testbp
  namespace: kanister
actions:
  backup:
    type: Namespace
    phases:
    - func: KubeTask
      name: test
      args:
        namespace: "{{ .Namespace.Name }}"
        image: kanisterio/kanister-tools:0.20.0
        command:
          - bash
          - -o
          - errexit
          - -c
          - |
            kando output "password" "{{ index .Secrets.postgres.Data "postgresql-password" | toString }}"
```
Secret:
```
apiVersion: v1
kind: Secret
metadata:
  name: pgsecret
type: Opaque
data:
  # value is "helloworld"
  postgresql-password: aGVsbG93b3JsZA==
```

ActionSet creation:
```
kanctl create actionset --action backup --blueprint testbp --namespacetargets kanister --profile s3-profile-4ml5t --secrets postgres=kanister/pgsecret
actionset backup-9wg2d created
```

ActionSet output:
```
ubectl get actionset backup-9wg2d -oyaml
apiVersion: cr.kanister.io/v1alpha1
kind: ActionSet
...
status:
  actions:
  - artifacts: null
    blueprint: testbp
    name: backup
    object:
      apiVersion: ""
      group: ""
      kind: namespace
      name: kanister
      namespace: kanister
      resource: ""
    phases:
    - name: test
      output:
        password: helloworld
      state: complete
  state: complete
```

**Comment by gouglhupf** [comment]
Thanks for fixing this.
When can I expect a new release with this fix?
